docs/SETUP.md

```
@@ -0,0 +1,456 @@
+If you see errors about missing environment variables, you missed a required API key. Check Step 3 again.
+
+---
+
+## 🚀 Step 5: Start Development Server
+
+```bash
+npm run dev
+```
+
+You should see output like:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)